source/features/update-pr-from-base-branch.tsx

```diff
@@ -49,16 +50,23 @@ async function handler({delegateTarget}: DelegateEvent): Promise<void> {
 async function addButton(position: Element): Promise<void> {
 	const {base, head} = getBranches();
 	const prInfo = await getPrInfo(base, head);
-	if (!prInfo) {
-		return;
-	}
 
-	if (prInfo.viewerCanEditFiles && prInfo.mergeable !== 'CONFLICTING') {
-		position.append(' ', (
-			<span className="status-meta d-inline-block rgh-update-pr-from-base-branch">
-				You can <button type="button" className="btn-link">update the base branch</button>.
-			</span>
-		));
+	if (prInfo.needsUpdate) {
+		if (pageDetect.isEnterprise()) {
```

Comment:
Why?

Reply:
Because they do not support `compare` yet

Reply:
We still have this information via v3 API, right?

Reply:
Your correct I did not realize that this information exists